src/proxies_v8/proxies_v8.ts

```diff
@@ -267,7 +267,7 @@ proxiesV8.use('/microsite/read/insights',
 
 proxiesV8.use('/content/aggregation/search',
   // tslint:disable-next-line: max-line-length
-  proxyCreatorSunbirdSearch(express.Router(), `${CONSTANTS.KONG_API_BASE}/content/aggregation/search`)
+  proxyCreatorSunbirdSearch(express.Router(), `${CONSTANTS.KONG_API_BASE}/aggregation/search`)
```

Comment:
I don't see any `${CONSTANTS.KONG_API_BASE}/aggregation/search -- which service it points too ?
`${CONSTANTS.KONG_API_BASE}/aggregation/search -- is very generic -- what is this regarding too...

Reply:
@karthik-tarento , This is regarding Top content - Microsites , It is pointing to cb- ext service.

Reply:
@karthik-tarento  This is related to top courses @dkttarento please made this url changes as understood what this is related to